### ISSUE TITLE
Not accessible with the screen readers

### ISSUE THREAD

**Issue by rahulbhai9** [Clariity/react-chessboard#171]
I am a blind person. So I daily use screen readers. This is a really nice library but unfortunately it is not accessible with the screen readers. It means that any website which uses this library will be not accessible for people with disability. It is really not difficult to fix this issue. We just need to add aria-label to squares. I am ready to help.

**Comment by Clariity** [comment]
Thank you for raising this! If you're able to raise a pull request for this then please do. Otherwise I will try to carve out some time when I am not busy but that will not be for a while

**Comment by Clariity** [comment]
Closing due to lack of activity
